src/main/java/uk/gov/hmcts/reform/unspec/stateflow/model/State.java

```diff
@@ -22,6 +26,10 @@ public String getName() {
         return name;
     }
 
+    public boolean isFlowState(FlowState flowState) {
```

Comment:
I would try to keep separation between `stateflow` - ( for core state flow that can be reused by others) and `flowstate` - ( where we will have our project specific things). is it possible to have it a utility method in `flowstate`?

Reply:
Removed the method for now, it's only used in single callback handler. Will add it to FlowState if it's needed in more than one place.